modules/highlight/highlight.go

```diff
@@ -39,6 +39,10 @@ func NewContext() {
 // Code returns a HTML version of code string with chroma syntax highlighting classes
 func Code(fileName, code string) string {
 	NewContext()
+	// don't highlight over 25kb
+	if len(code) > 25000 {
```

Comment:
This value should be moved to constant variable

Reply:
Maybe make a config option?

Reply:
Option can be also added in other PR, for this PR it would be fine also with constant

Reply:
Updated thanks!